### PR TITLE
Delete Member and Delete Event

### LINKED ISSUE #43: Make Delete Event functionality

### LINKED ISSUE #46: Make Delete Member functionality

### PR DESCRIPTION
**Pull request by SelwynAng**
Closes #46 and #43.

Made the following changes:
- Abstracted out `getTypicalAddressBook` methods from `TypicalEvent` and `TypicalMember` into a new class called `TypicalAddressBook`
- Deleted `DeleteCommand` and `DeleteCommandParser` classes
- Created 2 new command classes: `DeleteMemberCommand` and `DeleteEventCommand`
- Created 2 new parser classes: `DeleteMemberCommandParser` and `DeleteEventCommandParser`
- Created test cases for `DeleteMemberCommand`, `DeleteEventCommand`, `DeleteMemberCommandParser` and `DeleteEventCommandParser`.

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -23,6 +24,8 @@ public class ModelManager implements Model {
     private final UserPrefs userPrefs;
     private final FilteredList<Member> filteredMembers;
```
Comment (wr1159) [inline src/main/java/seedu/address/model/ModelManager.java]: nit but should we remove this line spacing

**Review by wr1159 — APPROVED** [review]
LGTM, just one small nit to be fixed

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (SelwynAng) [reply src/main/java/seedu/address/model/ModelManager.java]: Roger, will fix that!

File: src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        DeleteEventCommand deleteEventFirstCommandCopy = new DeleteEventCommand(INDEX_FIRST_EVENT);
+        assertTrue(deleteEventFirstCommand.equals(deleteEventFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(deleteEventFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(deleteEventFirstCommand.equals(null));
+
+        // different member -> returns false
```
Comment (dylkaw) [inline src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java]: Very minor typo.
```suggestion
        // different event -> returns false
```

**Review by dylkaw — APPROVED** [review]
LGTM! Only a small typo in one of the comments.

File: src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java (hunk 2 above)
Comment (SelwynAng) [reply src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java]: Alright, will correct this!

**Pull request merged**